### ISSUE TITLE
Quran and Hadith on fasting

### ISSUE THREAD

**Issue by kufarooq** [kufarooq/Towards-Islam#256]
<b>Qur&rsquo;an and Hadith on fasting</b><br><div dir="ltr" style="text-align: left;">

<div class="separator" style="clear: both; text-align: center;"></div>

<div class="separator" style="clear: both; text-align: center;"><a href="http://ift.tt/1PwPahJ" style="margin-left: 1em; margin-right: 1em;"><img border="0" height="295" src="http://ift.tt/1UqOA14" width="400"></a></div>

<div style="text-align: justify;">O you, who believe. Fasting is prescribed for you as it was prescribed for those before you, that you may become Al-Muttaqun (the pious). (Fasting) for a fixed number of days, but if any of you is ill or on a journey, the same number (should be made up) from other days. And as for those who can fast with difficulty, (i.e. an old man, etc.), they have (a choice either to fast or) to feed a poor person (for every day). But whoever does good of his own accord, it is better for him. And that you fast, it is better for you if only you know.&rdquo; (Qur&rsquo;an, 2:183-84)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">&ldquo;The month of Ramadan in which was revealed the Qur&rsquo;an, a guidance for mankind and clear proofs for the guidance and the criterion (between right and wrong). So whoever of you sights (the crescent on the first night of) the month (of Ramadan), he must fast that month, and whoever is ill or on a journey, the same number (of days which one did not fast must be made up) from other days. Allah intends for you ease, and He does not want to make things difficult for you. (He wants that you) must complete the same number (of days), and that you must magnify Allah for having guided you so that you may be grateful to Him&rdquo; (Qur&rsquo;an, 2:185).</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">&ldquo;It is made lawful for you to have sexual relations with your wives on the night of the fasts. They are garments for you and you are the same for them. Allah knows that you used to deceive yourselves, so He turned to you and forgave you. So now have sexual relations with them and seek that which Allah has ordained for you, and eat and drink until the white thread (light) of dawn appears to you distinct from the black thread (darkness of night), then complete your fast till the nightfall. And do not have sexual relations with them (your wives) while you are in Itikaf in the mosques. These are the limits (set) by Allah, so approach them not. Thus does Allah make clear His signs to mankind that they may become Al Muttaqun (the pious).&rdquo; (Qur&rsquo;an, 2:187)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">&ldquo;Verily! We have sent it (this Qur&rsquo;an) down in the night of Al-Qadr. And what will make you know what the night of Al-Qadr is? The night of Al-Qadr is better than a thousand months. Therein descend the angels and the Ruh (Gabriel) by Allah&rsquo;s Permission with all Decrees, Peace! Until the appearance of dawn.&rdquo; (Qur&rsquo;an, 97:1-5)</div>

<div class="separator" style="clear: both; text-align: center;"><a href="http://ift.tt/1PwOcSS" style="margin-left: 1em; margin-right: 1em;"><img border="0" height="245" src="http://ift.tt/1UqOk1T" width="400"></a></div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Hadiths on fasting</b></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace and blessings be upon him) said: &ldquo;Whoever fasts during Ramadan with faith and seeking his reward from Allah will have his past sins forgiven. Whoever prays during the nights in Ramadan with faith and seeking his reward from Allah will have his past sins forgiven. And he who passes Lailat Al-Qadr in prayer with faith and seeking his reward from Allah will have his past sins forgiven.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace be upon him) said: &ldquo;If anyone omits his fast even for one day in Ramadan without a concession or without being ill, then if he were to fast for the rest of his life he could not make up for it.&rdquo; (Bukhari)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace be upon him) said: Allah the Majestic and Exalted said: &ldquo;Every deed of man will receive ten to 700 times reward, except Siyam (fasting), for it is for Me and I shall reward it (as I like). There are two occasions of joy for one who fasts: one when he breaks the fast and the other when he will meet his Lord&rdquo; (Muslim).</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace be upom him) said: Many people who fast get nothing from their fast except hunger and thirst, and many people who pray at night get nothing from it except wakefulness (Darimi).</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>When to start fasting</b></div>

<div style="text-align: justify;">Ibn Umar related that the Prophet said: &ldquo;Do not start fasting unless you see the new moon, and do not end fasting until you see it. If the weather is cloudy then calculate when it should appear.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Suhoor (pre-dawn meal)</b></div>

<div style="text-align: justify;">Anas related that the Prophet said: &ldquo;Take the suhoor meal, for there is blessing in it.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Breaking your fast</b></div>

<div style="text-align: justify;">Salman ibn Amir Dhabi related that the Prophet said: &ldquo;Break your fast with dates, or else with water, for it is pure.&rdquo; (Abu Dawud, Tirmidhi)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>During journey</b></div>

<div style="text-align: justify;">Aisha related that the Prophet (peace be upon him) was asked whether one should fast when on a journey, and he replied: &ldquo;Fast if you like, or postpone it if you like.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Behavior while fasting</b></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace be upon him) said: &ldquo;If a person does not avoid false talk and false conduct during Siyam, then Allah does not care if he abstains from food and drink.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Forgetfully eating or</b></div>

<div style="text-align: justify;"><b>drinking while fasting</b></div>

<div style="text-align: justify;">Abu Huraira related that the Prophet (peace be upon him) said: &ldquo;If anyone forgets that he is fasting and eats or drinks he should complete his Siyam, for it is Allah who has fed him and given him drink.&rdquo; (Bukhari, Muslim)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Providing for those who are breaking the fast</b></div>

<div style="text-align: justify;">Zaid ibn Khalid Johni related that the Prophet (peace be upon him) said: &ldquo;He who provides for the breaking of the Siyam of another person earns the same merit as the one who was observing Siyam without diminishing in any way the reward of the latter.&rdquo; (Tirmidhi)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>Lailat Al-Qadr</b></div>

<div style="text-align: justify;">Aisha related that the Prophet (peace be upon him) said: &ldquo;Look for Lailat Al-Qadr on an odd-numbered night during the last 10 nights of Ramadan.&rdquo; (Bukhari).</div>

<div style="text-align: justify;">Anas ibn Malik related that the Prophet (peace be upon him) said: &ldquo;When Lailat Al-Qadr comes, Gabriel descends with a company of angels (may Allah bless them all) who ask for blessings on everyone who is remembering Allah, whether they are sitting or standing.&rdquo; (Baihaqi)</div>

<div style="text-align: justify;"><br></div>

<div style="text-align: justify;"><b>&nbsp;soundvision.com</b></div>

</div>
